### DISCUSSION TITLE
Congrats on v5.0

### DISCUSSION THREAD

**Discussion by Andrew-McGee** [ampache/ampache#3004] [General]
Well done Ampache devs for getting to v5. I know many of you put in many hours to contribute to this release so wanted to say thanks to all. 

I will probably move off develop and stick with release 5 for my front end now.

Cheers!

**Comment by Afterster** [comment]
Congrats @lachlan-00 as the main developer. Based on the changelog you may claim https://www.bountysource.com/issues/2987113-feature-request-ability-to-limit-specific-catalogues-to-users :)

**Reply by lachlan-00** [reply]
haha, it should hopefully get refunded seeing as i closed the original issue a few months ago, just a good bugbear to get off my back!

**Comment by lachlan-00** [comment]
this was a really good one to get done. so many major developments that will hopefully let us just extend instead of rewriting over and over again.

**Comment by kuzi-moto** [comment]
I'd like to ditto the congratulations! I have been following the progress of this project and quite happy to see the changes that are happening. I look forward to using Ampache again with Docker once I find time to reconfigure my server setup.

**Comment by vollmerk** [comment]
So glad to see this project chugging along :D and all the work on what I can tell was a massive rewrite :)

**Reply by vollmerk** [reply]
too clarify it might say "maintainer" by my name but I contributed nothing to this release! I'm just a cheerleader/custodian that passes the torch if and when it needs to be passed again :)

**Reply by lachlan-00** [reply]
that's also a definition of a maintainer

**Reply by vollmerk** [reply]
Hahah ok you've got me there.. :D Great job btw :)
